provisioning-workflows/zOS/NetworkResources/Provision_Network_Resource.xml

```diff
@@ -0,0 +1,252 @@
+        <abstract>Obtain APPLID flag</abstract>
+        <description>Flag to indicate whether or not to obtain APPLID resource</description>
+        <category>Boolean Variables</category>
+        <string valueMustBeChoice="true">
+            <choice>YES</choice>
+            <choice>NO</choice>
+        </string>
+    </variable>
+    <variable name="IP_ADDRESS" visibility="public">
+        <label>IP_ADDRESS</label>
+        <abstract>TCP/IP DVIPA IP Address</abstract>
+        <description>TCP/IP DVIPA IP Address</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="IP_ADDRESS_ID" visibility="private">
+        <label>IP_ADDRESS_ID</label>
+        <abstract>Object ID of the IP Address resource</abstract>
+        <description>The object id of the allocated IP address to use during deprovisioning</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="PORT" visibility="public">
+        <label>PORT</label>
+        <abstract>TCP/IP Port Number</abstract>
+        <description>TCP/IP port number</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="PORT_ID" visibility="private">
+        <label>PORT_ID</label>
+        <abstract>Object ID of the port resource</abstract>
+        <description>The object id of the reserved Port address to use during deprovisioning</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="APPLID" visibility="public">
+        <label>APPLID</label>
+        <abstract>VTAM APPLID</abstract>
+        <description>VTAM APPLID</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="APPLID_ID" visibility="private">
+        <label>APPLID_ID</label>
+        <abstract>Object ID of the APPLID resource</abstract>
+        <description>The object id of the reserved APPLID resource to use during deprovisioning</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="JOB_NAME" visibility="public">
+        <label>JOB_NAME</label>
+        <abstract>Job name of server that will use allocated port. Specify * if Jobname is not available</abstract>
+        <description>Job name of server that will use allocated port. Specify * if Jobname is not available</description>
+        <category>REST</category>
+        <string valueMustBeChoice="false" multiLine="false"/>
+    </variable>
+    <variable name="USAGE_TYPE" visibility="public">
+        <label>USAGE_TYPE</label>
+        <abstract>Usage type that matches with usage type associated with port pool</abstract>
+        <description>Usage type that matches with usage type associated with port pool. Request will fail if matching usage type is not found</description>
+        <category>REST</category>
+        <string valueMustBeChoice="false" multiLine="false"/>
```

Comment:
Are there any example standard choices that we could provide here?

Reply:
We won't be able to provide choice here because it depends on how network admin has setup the port ranges and what usage_type is defined there